doc/rst/source/movie.rst

```diff
@@ -370,12 +370,26 @@ Other Movie Formats
 -------------------
 
 As configured, **movie** only offers the MP4 and WebM formats for movies.  The conversion is performed by the
-tool ffmpeg (https://www.ffmpeg.org), which has more codecs and processing options than there are children in China.
+tool FFmpeg (https://www.ffmpeg.org), which has more codecs and processing options than there are children in China.
 If you wish to run ffmpeg with other options, select mp4 and run **movie** with long verbose (**-Vl**).
 At the end it will print the ffmpeg command used.  You can copy, paste, and modify this command to
 select other codecs, bit-rates, and arguments.  You can also use the PNG sequence as input to tools such
 as QuickTime Pro, iMovie, MovieMaker, and similar commercial programs to make a movie that way.
 
+Manipulating Multiple Movies
+----------------------------
+
+If you are making a series of similar movies, you can use ffmpeg to paste and stitch them into a single movie.
+Assume we have four movies called movie_1.mp4, movie_2.mp4, movie_3.mp4, and movie_4.mp4, and you wish to combine
+them into a 2x2 panel showing the four movies simultaneously.  You would first combine movies (1,2) and (3,4)
```

Comment:
Shouldn't it be ``combine movies (0,1) and (2,3)``?

Reply:
No, those numbers are not under GMT control, it is just what the user (here me) named the final movies.

Reply:
😇